packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -4,10 +4,16 @@ import { Item } from '@dailydotdev/react-contexify';
 import Link from 'next/link';
 import AuthContext from '../contexts/AuthContext';
 import PowerIcon from './icons/Power';
-import { UserIcon } from './icons';
+import { KeyReferralIcon, UserIcon } from './icons';
 import DevCardIcon from './icons/DevCard';
 import SettingsIcon from './icons/Settings';
 import { IconSize } from './Icon';
+import { useLazyModal } from '../hooks/useLazyModal';
+import { LazyModal } from './modals/common/types';
+import TimerIcon from './icons/Timer';
```

Comment:
Should also be from the main index export

Reply:
Cool. Let me add it. Think this was a legacy one and was not included in the index import.